@types/@zondax/filecoin-signing-tools/index.d.ts

```diff
@@ -0,0 +1,241 @@
+declare module '@zondax/filecoin-signing-tools' {
```

Comment:
Where are these `.d.ts` files coming from? Did you write them yourself?

It would be great to eventually contribute these typings to `@zondax/filecoin-signing-tools` to make them useful for the wider community and so that we don't have to maintain them on our own.

Maybe we can open a follow-up issue in our repo for this task?

Reply:
I copied these over from https://github.com/glifio/react-components/tree/primary/%40types/%40zondax, that repo also being the reason we're using this module in the first place.

Good call, I opened https://github.com/filecoin-station/filecoin-station/issues/402